src/code/z_actor.c

```diff
@@ -2839,10 +2840,11 @@ void Actor_DrawAll(PlayState* play, ActorContext* actorCtx) {
 
             actor->isDrawn = false;
             if ((actor->init == NULL) && (actor->draw != NULL) && (actor->flags & actorFlags)) {
-                if ((actor->flags & ACTOR_FLAG_80) &&
-                    ((play->roomCtx.curRoom.unk5 == 0) || (play->actorCtx.lensMaskSize == LENS_MASK_ACTIVE_SIZE) ||
+                if ((actor->flags & ACTOR_FLAG_REACT_TO_LENS) &&
+                    ((play->roomCtx.curRoom.lensMode == LENS_MODE_HIDE_ACTORS) ||
```

Comment:
Hmm, this is probably actually a boolean given that it's only compared to 0, but that sounds dreadful to try to name.

Reply:
yeah we went the lets-not-make-this-a-bool route in OoT after failing to find a good name for it as a bool